src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,40 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static final long LAST_BOOK = 3L;
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
+        Book book = new Book();
+        book.setTitle("Thinking in Java");
+        book.setPrice(BigDecimal.valueOf(19.99));
+        bookDao.create(book);
+        System.out.println(book);
+
+        book = bookDao.findById(LAST_BOOK).orElse(null);
```

Comment:
The use of orElse(null) when retrieving a book by ID can lead to NullPointerExceptions if the book is not found. Consider using orElseThrow or checking if the Optional is present before proceeding.